catalogs/catalog-jdbc-mysql/src/test/java/org/apache/gravitino/catalog/mysql/operation/TestMysqlTableOperations.java

```diff
@@ -64,7 +64,7 @@ public void testOperationTable() {
             .withName("col_1")
             .withType(VARCHAR)
             .withComment("test_comment")
-            .withNullable(true)
+            .withNullable(false)
```

Comment:
Why do we change this column?

Reply:
it is used as a unique index at 93 line use

Reply:
Got it. Thanks.